tools/sggolangcilint/tools.go

```diff
@@ -96,5 +96,14 @@ func PrepareCommand(ctx context.Context) error {
 	); err != nil {
 		return fmt.Errorf("unable to download %s: %w", name, err)
 	}
+
+	configPath := defaultConfigPath()
+	if err := os.MkdirAll(filepath.Dir(configPath), 0o755); err != nil {
+		return err
+	}
+	if err := os.WriteFile(configPath, DefaultConfig, 0o600); err != nil {
+		return err
+	}
```

Comment:
Can we not avoid doing this on the `PrepareCommand` which is usually used for installing the binary?

Reply:
I'd like the config to be present when running `CommandInDirectory`. It used to be written in `Run` every time that was called, but I felt like that was a bit excessive (and calling it every time in `CommandInDirectory` feels even more excessive).

Do you have any suggestions where to write this instead? I'd like it to be written at least often enough so that changes to it is reflected, so simply checking for existence seems not enough. 🤔

Reply:
I think that's reasonable. Let's keep it like that